hive-metastore/src/main/java/org/apache/iceberg/hive/HiveOperationsBase.java

```diff
@@ -123,13 +185,14 @@ default void persistTable(Table hmsTable, boolean updateHiveTable, String metada
     }
   }
 
-  static StorageDescriptor storageDescriptor(TableMetadata metadata, boolean hiveEngineEnabled) {
```

Comment:
same as above, we need to keep `StorageDescriptor storageDescriptor(TableMetadata metadata, boolean hiveEngineEnabled)` but it can internally call `static StorageDescriptor storageDescriptor(Schema schema, String location, boolean hiveEngineEnabled)`